README.md

```diff
@@ -1 +1,74 @@
-# NewAwesome-book
```

Comment:
- [ ] There is another thing that needs to be fixed that you should have a folder called `modules`. I would recommend changing your `js` folder to `modules` to fix this issue.  You could also add more modules for smaller functions to make your code better 👋🍀

See this image
![image](https://user-images.githubusercontent.com/69011963/148746674-60edd22b-7319-45ab-895d-66658b67cbe3.png)